src/schema/notifications.ts

```diff
@@ -422,6 +423,22 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         });
       }
 
+      const newCommentComments = data.filter(({ notificationType }) =>
```

Comment:
I know its not a part of this pr but just noticed that user can send unrestricted amount of `data` here, can we do just a quick `if (data.length) > 100) throw`, to avoid doing large `IN()` statements?

Reply:
(also fine to just open a task for this)

Reply:
https://dailydotdev.atlassian.net/browse/AS-673

Reply:
I also fixed this one :)